frb_codegen/.clippy.toml

```diff
@@ -0,0 +1,2 @@
+# TOOD: Remove this after refactoring
+too-many-lines-threshold = 150
```

Comment:
uber-nit: Is it to be fixed in this PR or future? If the latter we can create an issue in the issues section to avoid forgetting it :)

Reply:
Created #661 to resolve this issue.